### ISSUE TITLE
False positiv in no-sequences

### ISSUE THREAD

**Issue by ptitgraig** [eslint/eslint#1937]
Hello.

I'm wondering wether that's a false positiv or not. 
I've run the following code against eslint and it raised according to the rule `no-sequences`. 

```javascript
var found, index;
found = false,
index = 0;
```

Do you think that's a false positiv ?

Regards

**Comment by gyandeeps** [comment]
This is considered as a warning. So the rule is correctly flagging it.
Docs mentions your use case under the warning section.

**Comment by nzakas** [comment]
Can you paste in the errors you're seeing?

**Comment by ptitgraig** [comment]
Answering Nicholas.
```shell
 699:16   warning  Unexpected use of comma operator                                       no-sequences
```

Related code (same as the one reported on #1936 ):

```
isShiftTab: function(arr, ele) {
  var found, index;
  found = false,
  index = 0;
  while (!found && index < arr.length) {
    if (arr[index] === ele) {
      found = true;
    } else {
      index++;
    }
  }
  return found;
},
```

Regards.

**Comment by nzakas** [comment]
A sequence is when a comma is used outside of a variable declaration.

```js
  found = false,
  index = 0;
```

That code is a sequence. You can remove the warning by changing the commas to a semi colon:

```js
  found = false;
  index = 0;
```
